core/src/main/java/org/neo4j/graphalgo/core/heavyweight/HeavyCypherGraphFactory.java

```diff
@@ -280,15 +279,18 @@ public boolean visit(Result.ResultRow row) throws RuntimeException {
                 }
                 if (hasRelationshipWeights) {
                     long relId = RawValues.combineIntInt(source, target);
-                    relWeigths.set(relId, row.get("weight"));
+                    Object weight = getProperty( row, "weight" );
```

Comment:
Since we did not adopt the Neo4j style guide, we might end up in a couple of format-heavy PR change sets here or there.

Reply:
Yeh sorry about that. I'll use the default Intellij formatting settings when working on this project so that everything stays consistent.